.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: objective-c
-osx_image: xcode7.1
+osx_image: xcode8
 xcode_workspace: YelpAPI.xcworkspace
 xcode_scheme: YelpAPI
-xcode_sdk: iphonesimulator9.1
+xcode_sdk: iphonesimulator
```

Comment:
If you don't specify a version number, like this, it uses the latest version of the SDK. That matches what's in our project file (where we ask to build against the latest available version). basher made this change to YLTableView in Yelp/YLTableView#16, and I didn't want to keep updating the version number so it seemed like a good idea 😛